collectors/ebpf.plugin/multi_metadata.yaml

```
@@ -4,21 +4,21 @@ modules:
       plugin_name: ebpf.plugin
       module_name: filedescriptor
       monitored_instance:
-        name: ebpf filedescriptor
+        name: Filedescriptor
         link: ""
         categories:
           - data-collection.ebpf
-        icon_filename: ""
+        icon_filename: "ebpf.jpg"
       related_resources:
         integrations:
           list: []
       info_provided_to_referring_integrations:
         description: ""
-      keywords: []
+      keywords: ["file", "eBPF"]
```

```suggestion
      keywords:
        - file
        - eBPF
```

Regular YAML lists are fine here, and are generally preferred because it makes diffs cleaner when the list contents need to be changed.

---

Thank you very much, last commit fixed the confusion.